### ISSUE TITLE
[BUG] Strange Behavior With Long Scroll Bars

### ISSUE THREAD

**Issue by PoppyBlossom** [ClanGenOfficial/clangen#2618]
**Type**: (only select one)
- UI - anything to do with the user interface. maybe a button is not working, or the way some assets are formatted seems to be off. 

**Describe the bug**
Strange behavior when scrolling long scroll bars

**Grade**: (only select one)
- Regular - A typical bug that should be seen to soon, but doesn’t necessarily break the game or inhibit the player

**Additional Contexts**
_I can't fix this because I have no idea how UI works in pygame_
Thank you @Vickarooni for the save file.(#2597)

**Reproduce**
1. Find an extremely long scroll bar
2. Scroll down, at some point, it will gradually be harder to scroll.

**Commit # or Game Version Number**: 
7b48f10b

**Screenshots**

https://github.com/user-attachments/assets/a867f3f4-d33d-4846-8ed0-e164b31af630

**Comment by scribblecrumb** [comment]
Oh, wild, I thought I had this fixed, I'll have to look into it again

**Comment by PoppyBlossom** [comment]
The scrollbar in #2503 is also problematic.

**Comment by Vickarooni** [comment]
pfft yw for the save haha. i also saw it on another one of ur PRs and i was like "wait i know that save" LMAO

**Comment by j-gynn** [comment]
Seems to no longer be a problem as of the latest version? @PoppyBlossom can you confirm if this still causes an issue?

**Comment by scribblecrumb** [comment]
Still seems to be an issue.  Though genuinely don't know how to further fix this.  Might just be a limitation that we bring to py-gui's attention.

**Comment by j-gynn** [comment]
I think it's a quirk of floating-point error, once your scrollable box gets to a certain size it literally can't be accurate any more.

Suggestion: we could look into chunking with infinite scroll?

**Comment by scribblecrumb** [comment]
Yeah I wondered if that was the case.  And sure! If you have ideas for that, I'd love to see what you come up with.  My only other suggestion was to implement some pagination if it gets too long, but that's not particularly ideal.
